src/openforms/formio/service.py

```diff
@@ -53,12 +56,23 @@ def update_configuration_for_request(configuration: dict, request: Request) -> N
 
     The configuration is modified in the context of the provided :arg:`submission`.
     """
-    update_urls_in_place(configuration, request)
+    pipeline = (
+        update_urls_in_place,
+        update_default_file_types,
+    )
```

Comment:
A pattern emerges here! But we already have a pattern: Can you refactor this into a register and register these "formatters" like here: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/formio/formatters/service.py  and https://github.com/open-formulieren/open-forms/blob/master/src/openforms/formio/formatters/default.py

We probably will have more to come!